local-dev/compile-proto.sh

```diff
@@ -4,11 +4,31 @@ set -e
 # define variable
 SCRIPT_DIR=$(cd "$(dirname "$0")" && pwd)
 WORK_DIR=$(cd "$SCRIPT_DIR/.." && pwd)
-PROTOC_DOCKER_DIR=$(cd "$SCRIPT_DIR/../docker/protoc" && pwd)
+PUBLIC_PROTOS=$(ls "$WORK_DIR"/schemas | grep -v -E "^internal")
+INTERNAL_PROTOS=$(ls "$WORK_DIR"/schemas/internal)
+docker_run="docker run --rm -it -v ${WORK_DIR}:/littlehorse protoc:23.4"
 
 # compile protoc
-echo "Compiling protoc" $(docker build -q --file ${PROTOC_DOCKER_DIR}/Dockerfile --tag protoc ${PROTOC_DOCKER_DIR})
-echo "Protoc version" $(docker run --rm -it protoc protoc --version)
+echo "Compiling docker image 'protoc:23.4'"
+docker build -q --tag protoc:23.4 -<<EOF
```

Comment:
can we tag it `lh-protoc` so that when people do `docker images` they can see it's littlehorse?

Reply:
sure